nixos/modules/services/cluster/k3s/default.nix

```
@@ -434,6 +434,21 @@ in
         for further information.
       '';
     };
+
+    extraKubeProxyConfig = lib.mkOption {
+      type = with lib.types; attrsOf anything;
+      default = { };
+      example = {
+        mode = "nftables";
+      };
```

Sorry, one last thing. As users are expected to set clientConnection.kubeconfig`, it would make sense to show this in the example. Otherwise the example would be an invalid configuration.

```suggestion
      example = {
        mode = "nftables";
        clientConnection.kubeconfig = "/var/lib/rancher/k3s/agent/kubeproxy.kubeconfig";
      };
```

---

oh i forgot to update example, thanks.